docs/src/api/class-worker.md

```diff
@@ -52,6 +52,12 @@ foreach(var pageWorker in page.Workers)
 }
 ```
 
+**addInitScript for Worker**
+
+Currently, methods like [`method: Page.addInitScript`] is not supported for worker. See issue [#28029](https://github.com/microsoft/playwright/issues/28029).
+
+However, there is a workaround by routing the main worker script. For code example, see [this comment](https://github.com/microsoft/playwright/issues/28029#issuecomment-1802788239). For full repository demo, see [this repository](https://github.com/doehyunbaek/monkeypatch-worker).
```

Comment:
Instead of linking to issues/repos, we prefer to make an inline code block with an example right here.

Reply:
I guess we should provide code examples for Node.js / Python / Java / .Net? As I have never used other apis other than Node.js, I think it is going to take a little bit more time to get the code blocks for other three languages.